src/Console/Exporters/FluentExporter.php

```diff
@@ -414,44 +453,44 @@ private function exportAssociations(array $associationMapping)
                     break;
 
                 case ClassMetadataInfo::FETCH_EAGER:
-                    $fetch = "->fetchEager()";
+                    $fetch = '->fetchEager()';
                     break;
 
                 case ClassMetadataInfo::FETCH_EXTRA_LAZY:
-                    $fetch = "->fetchExtraLazy()";
+                    $fetch = '->fetchExtraLazy()';
                     break;
             }
         }
 
         $joinColumn = '';
         if ($joinColumns) {
             foreach ($joinColumns as $c) {
-                $columnName           = $c['name'];
+                $columnName = $c['name'];
                 $referencedColumnName = $c['referencedColumnName'];
-                $nullable             = !isset($c['nullable']) || $c['nullable'] == true ? 'true' : 'false';
-                $unique               = !isset($c['unique']) || $c['nullable'] == false ? 'false' : 'true';
-                $onDelete             = isset($c['onDelete']) ? '\'' . $c['onDelete'] . '\'' : 'null';
-                $columnDefinition     = isset($c['columnDefinition']) ? '\'' . $c['columnDefinition'] . '\'' : 'null';
+                $nullable = !isset($c['nullable']) || $c['nullable'] == true ? 'true' : 'false';
```

Comment:
Is this behaviour correct? If `$c['nullable']` is not set this will evaluate `true` but isn't doctrine default behaviour to be non null? @patrickbrouwers